tests/test_alloc.rs

```diff
@@ -1,19 +1,22 @@
 use alloc_counter::AllocCounterSystem;
+use deku::ctx::Endian;
```

Comment:
Do we want to add this to the prelude? (I'm debating, I'm leaning more towards no as users typically won't be needing it)

Reply:
I agree with you. IMO, most of users write codes like:
```rust
#[derive(DekuRead, DekuWrite)]
struct DataType {
    // ...
}
let data = DataType::from_bytes(input);
data.to_bytes();
```
In most case they don't need to care about context.